vitest.edge.config.ts

```diff
@@ -0,0 +1,25 @@
+import { defineConfig } from 'vitest/config';
+
+/**
+ * Vitest configuration for Edge runtime testing.
+ *
+ * This config runs all tests EXCEPT the express subfolder, which contains
+ * Node.js-specific code (Express framework) that is not compatible with
+ * Edge runtime environments.
+ *
+ * The Edge runtime is provided by @edge-runtime/vm, which emulates
+ * Cloudflare Workers, Vercel Edge Functions, and similar environments.
+ */
+export default defineConfig({
+  test: {
+    globals: false,
+    environment: 'edge-runtime',
+    include: ['test/**/*.spec.ts'],
+    exclude: [
+      // Express tests require Node.js-specific APIs (http, Express framework)
+      'test/server/a2a_express_app.spec.ts',
+      // Node modules should always be excluded
+      '**/node_modules/**',
+    ],
+  },
+});
```

Comment:
How about

```suggestion
export default mergeConfig(defaultConfig, {
  test: {
    environment: 'edge-runtime',
    exclude: [
      // Express tests require Node.js-specific APIs (http, Express framework)
      'test/server/a2a_express_app.spec.ts',
      // Node modules should always be excluded
      '**/node_modules/**',
    ],
  },
});
```

and also imports are going to be

```ts
import { mergeConfig } from 'vitest/config';
import defaultConfig from './vitest.config';
```

Currently we don't have a lot to inherit but just to avoid changing two places in the future if we have more.

Reply:
Ah I complete forgot about mergeConfig. Added 👍